From 2e0533533c5bc7a0aba00c4f3886f03d4d000eb6 Mon Sep 17 00:00:00 2001
From: Murala S Swamy <ssmurala@habana.ai>
Date: Thu, 23 Jan 2025 11:15:25 +0200
Subject: [PATCH] [FSW-12120] fs1: use mem io from embedded specs

With this patch, added support to use common
mem read write functionality from embedded specs

Change-Id: Ib22a14e3b870cbcc89a82425cfc336c1f4a00beb
---

diff --git a/arch/fs1/common.c b/arch/fs1/common.c
index 2d03d5c..ab64d16 100644
--- a/arch/fs1/common.c
+++ b/arch/fs1/common.c
@@ -23,6 +23,7 @@
 #include <hl_platform_defs.h>
 #else
 #include <baremetal.h>
+#include <mem_io.h>
 #include <device.h>
 #include <bitops.h>
 #include <pll.h>
diff --git a/lib/common/pci.c b/lib/common/pci.c
index b2bf281..eabc503 100644
--- a/lib/common/pci.c
+++ b/lib/common/pci.c
@@ -10,6 +10,7 @@
 #include <zephyr/drivers/misc/common/common_utils.h>
 #else
 #include <baremetal.h>
+#include <mem_io.h>
 #include <device.h>
 #include <bitops.h>
 #endif
